apps/fortuna/src/command/run.rs

```diff
@@ -223,6 +232,16 @@ async fn setup_chain_state(
         )
     })?;
 
+    let last_prior_commitment = provider_commitments.last();
```

Comment:
drive-by bugfix: the previous implementation would happily let you specify commitments for future sequence numbers. That of course makes no sense and indicates a configuration problem.